pkg/collector/corechecks/containers/docker/check_linux_test.go

```diff
@@ -286,3 +286,23 @@ func TestDockerNetworkExtension(t *testing.T) {
 	mockSender.AssertMetric(t, "Rate", "docker.net.bytes_rcvd", 7, "", []string{"foo:bar", "docker_network:bridge"})
 	mockSender.AssertMetric(t, "Rate", "docker.net.bytes_sent", 7, "", []string{"foo:bar", "docker_network:bridge"})
 }
+
+func TestNetworkCustomOnFailure(t *testing.T) {
+	// Make sure we don't panic if generic part fails
```

Comment:
Should we clarify somewhere that this means that `.PreProcess` was not called? Just to make the intent of the test clearer.

Reply:
It could be any of the `Pre|Process|Post` that was not called so I did not specify anything